README.md

```diff
@@ -212,6 +214,10 @@ class HelloWorldSimulation extends ConfigurationDrivenSimulations {
 
 `parts` is the list of parts that combined create your journey
 
+`run-if` is a list of labels. Runs this journey only if a label from this list is passed in the `labels` parameter of application.conf
+
+`skip-if` is a list of labels. Skips this journey if a label from this list is passed in the `labels` parameter of application.conf
+
```

Comment:
Should we mention that `run-if` and `skip-if` are optional? I see they haven't mentioned it for feeder. What do you think?

Reply:
Excellent point. I've added another commit with updated wording.